src/components/Icon/Icon.tsx

```diff
@@ -5,68 +5,149 @@ import svg4everybody from 'svg4everybody';
 import styles from './Icon.module.css';
 import icons from '../../icons/spritemap/spritemap.svg';
 
-export interface Props {
+interface IconPropsBase {
   /**
    * CSS class names that can be appended to the component.
    */
   className?: string;
   /**
-   * Declares whether the icon is focusable or not for accessibility
+   * Either a fragment of svg elements (g, path, circle, etc.), or a single
+   * svg element. Useful for creating specific icon components.
+   *
+   * @example
+   * function CircleIcon(props: IconProps) {
+   *   return (
+   *     <Icon {...props}>
+   *       <circle cx="50" cy="50" r="50" />
+   *     </Icon>
+   *   )
+   * }
    */
-  focusable?: boolean;
+  children?: React.ReactNode;
+  /**
+   * The SVG Color, expects a valid css color (hex, rgb, etc.).
+   * Recommendation: if `currentColor` isn't sufficient, use color tokens from
+   * @chanzuckerberg/eds/lib/tokens/colors.ts
+   */
+  color?: string;
+  /**
+   * If true, the element will be displayed as a block, otherwise
+   * default is inline like images
+   */
+  fullWidth?: boolean;
   /**
    * ID used so the svg can read the title of the SVG icon to the user when accessibility is needed
    */
-  id?: any;
+  id?: string;
+  /**
+   * Inverted variant for dark backgrounds
+   *
+   * TODO: investigate if needed
+   */
+  inverted?: boolean;
```

Comment:
I really don't think so, especially since you have the `color` prop now and devs can just pass in white. Also I talked to Sean about inverted buttons and we agreed there is no use case for them in LP at the moment or the foreseeable future; I think that applies to inverted icons and text too.